### ISSUE TITLE
Search Result Announcement After Enter

### ISSUE THREAD

**Issue by fhanif16** [tiles-lab/accessible-whiteboarding#6]
**# Expected Behavior**
When a user types a search term and presses Enter, the screen reader should announce the search results or say what was found.

**# Actual Behavior**
The screen reader does not announce any search results. The user has to move through the page manually to find out if any results are available.

**# Steps to Replicate**
1.Open the application.
2.Go to the search tool.
3.Type a search term.
4.Press Enter.
5.Notice that the screen reader does not announce the results.

**Comment by yousrahanif** [comment]
Screen reader still does not announce the total number of search results found
